vizro-core/changelog.d/20240325_170005_jo_stichbury_add_vale_styles.md

```diff
@@ -0,0 +1,48 @@
+<!--
```

Comment:
I think you can remove the two changelog files here, because you haven't made any changes in vizro-ai or vizro-core, so CI shouldn't complain if there are no changelog files added :)

Reply:
Done!